### PR TITLE
docs: vmanomaly - add component interaction diagram

### PR DESCRIPTION
**Pull request by fred-navruzov**
- small docs fixes
- add interaction diagram for vmanomaly components

image

### CODE REVIEW

File: docs/anomaly-detection/components/README.md [hunk 1]
```diff
@@ -21,5 +21,10 @@ This chapter describes different components, that correspond to respective secti
...
 
+> **Note**: starting from [v1.7.0](../CHANGELOG.md#v172), once the service starts, automated config validation is performed. Please see container logs for errors that need to be fixed to create fully valid config, visiting sections above for examples and documentation.
 
-> **Note**: starting from [v1.7.0](../CHANGELOG.md#v172), once the service starts, automated config validation is performed. Please see container logs for errors that need to be fixed to create fully valid config, visiting sections above for examples and documentation.
\ No newline at end of file
+Below, you will find an example illustrating how the components of `vmanomaly` interact with each other and with a single-node VictoriaMetrics setup.
+
+> **Note**: [Reader](/anomaly-detection/components/reader.html#vm-reader) and [Writer](/anomaly-detection/components/writer.html#vm-writer) also support [multitenancy](/Cluster-VictoriaMetrics.html#multitenancy), so you can read/write from/to different locations - see `tenant_id` param description.
+
+<img alt="vmanomaly-components" src="vmanomaly-components.webp" width="50%"/>
```
Comment (tenmozes) [inline docs/anomaly-detection/components/README.md:30]: why 50%? please use 800px

File: docs/anomaly-detection/components/README.md [hunk 2]
```diff
@@ -21,5 +21,10 @@ This chapter describes different components, that correspond to respective secti
 - [Writer section](writer.html) - Required
 - [Monitoring section](monitoring.html) -  Optional
 
+> **Note**: starting from [v1.7.0](../CHANGELOG.md#v172), once the service starts, automated config validation is performed. Please see container logs for errors that need to be fixed to create fully valid config, visiting sections above for examples and documentation.
```
Comment (tenmozes) [inline docs/anomaly-detection/components/README.md:24]: do we have anywhere explanation of labels for this metrics, like `model` and that we copy all other labers from original timeseries, if not we need to add it somewhere

File: docs/anomaly-detection/components/README.md (hunk 1 above)
Comment (fred-navruzov) [reply docs/anomaly-detection/components/README.md:30]: thanks, fixed

File: docs/anomaly-detection/components/README.md (hunk 2 above)
Comment (tenmozes) [reply docs/anomaly-detection/components/README.md:24]: please use abosulte pathers for link - like /anomaly-detection/change... or docs.victoriametrics.com/anomaly-detection/cha...

Comment (fred-navruzov) [reply docs/anomaly-detection/components/README.md:24]: nice catch, this was left from initial v1 commits, fixed

Comment (fred-navruzov) [reply docs/anomaly-detection/components/README.md:24]: > do we have anywhere explanation of labels for this metrics, like `model` and that we copy all other labers from original timeseries, if not we need to add it somewhere

1) https://docs.victoriametrics.com/anomaly-detection/components/writer.html#writer - right in the starting chapter (however, I will highlight it more explicitly with formatting)
2) the same page - https://docs.victoriametrics.com/anomaly-detection/components/writer.html#metrics-will also highlight it more explicitly with formatting
3) in guide - https://docs.victoriametrics.com/anomaly-detection/guides/guide-vmanomaly-vmalert.html#9-model-results
4) in FAQ - https://docs.victoriametrics.com/anomaly-detection/FAQ.html#what-is-anomaly-score , note section

**Review by tenmozes — APPROVED** [review]

**Pull request merged**